nuget/spec/fixtures/nuspecs/Microsoft.Extensions.DependencyModel.1.0.0.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Extensions.DependencyModel</id>
+    <version>1.0.0</version>
+    <authors>Microsoft.Extensions.DependencyModel</authors>
+    <owners>Microsoft.Extensions.DependencyModel</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/dotnet/core-setup/blob/master/LICENSE.TXT</licenseUrl>
+    <projectUrl>https://dot.net/</projectUrl>
+    <description>Abstractions for reading `.deps` files.</description>
+    <copyright>© Microsoft Corporation.  All rights reserved.</copyright>
+    <serviceable>true</serviceable>
+    <repository type="git" url="git://github.com/dotnet/core-setup" />
+    <dependencies>
+      <group targetFramework=".NETFramework4.5.1">
```

Comment:
I trimmed dependencies of this package out because they were causing a lot of extra nuget calls which didn't matter to our scenario.